Documentation/ceph-csi-drivers.md

```diff
@@ -108,105 +121,110 @@ Once the plugin is successfully deployed, test it by running the following examp
 
 # Test RBD CSI Driver
 
-## Create RBD StorageClass
-
-This
-[storageclass](../cluster/examples/kubernetes/ceph/csi/example/rbd/storageclass.yaml)
-expects a pool named `rbd` in your Ceph cluster. You can create this pool using
-[rook pool
-CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
-
-Please update `monitors` to reflect the Ceph monitors.
+## Create a CephBlockPool for our storage class to provision PVs from
 
-```console
-kubectl create -f cluster/examples/kubernetes/ceph/csi/example/rbd/storegeclass.yaml
+```bash
+cat << EOF | kubectl create -f -
```

Comment:
Isn't this cat redundant? IOW `kubectl create -f - <<EOF ...`

Reply:
Yup, I'll remove it